config/cardano/node.go

```diff
@@ -69,9 +70,13 @@ func NewCardanoNodeConfigFromFile(file string) (*CardanoNodeConfig, error) {
 }
 
 func (c *CardanoNodeConfig) loadGenesisConfigs() error {
+	prefix := "/"
 	// Load Byron genesis
 	if c.ByronGenesisFile != "" {
-		byronGenesisPath := path.Join(c.path, c.ByronGenesisFile)
+		byronGenesisPath := c.ByronGenesisFile
+		if !strings.HasPrefix(byronGenesisPath, prefix) {
```

Comment:
There's the `path/filepath.IsAbs()` function for this

Reply:
Neat. Switched to it.